project/plugins.sbt

```diff
@@ -1,6 +1,9 @@
 Compile / unmanagedSourceDirectories +=
   (ThisBuild / baseDirectory).value.getParentFile /
     "plugin" / "src" / "main" / "scala"
+Compile / unmanagedSourceDirectories +=
+  (ThisBuild / baseDirectory).value.getParentFile /
+    "plugin" / "src" / "main" / "scala-2.12"
```

Comment:
hmm, not sure I understand what this is for. doesn't sbt normally pick up a `scala-2.12` source directory without having to be asked...? but we're in the metabuild here, but we're reaching out to the main build...? I'm confused

Reply:
This is in `project/plugins.sbt`, which implements a trick to use plugin in its own repo, also known as "intertwined" dependency after this question https://stackoverflow.com/q/37424513/3827. Here, I'm adding `scala-2.12` directory to the metabuild, since I added Scala-specific sources to it.